dashboard/app/models/pd/teacher_application.rb

```diff
@@ -2,19 +2,19 @@
 #
 # Table name: pd_teacher_applications
 #
-#  id              :integer          not null, primary key
-#  created_at      :datetime         not null
-#  updated_at      :datetime         not null
-#  user_id         :integer          not null
-#  primary_email   :string(255)      not null
-#  secondary_email :string(255)      not null
-#  application     :text(65535)      not null
+#  id             :integer          not null, primary key
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  user_id        :integer          not null
+#  personal_email :string(255)      not null
+#  school_email   :string(255)      not null
+#  application    :text(65535)      not null
 #
 # Indexes
 #
-#  index_pd_teacher_applications_on_primary_email    (primary_email)
-#  index_pd_teacher_applications_on_secondary_email  (secondary_email)
-#  index_pd_teacher_applications_on_user_id          (user_id) UNIQUE
+#  index_pd_teacher_applications_on_personal_email  (personal_email)
+#  index_pd_teacher_applications_on_school_email    (school_email)
+#  index_pd_teacher_applications_on_user_id         (user_id) UNIQUE
```

Comment:
OK. I've limited this PR to schema and model changes relevant to the change I'm making.